ONSdigital/design-system
# Timeout panel component

### What is the context of this PR?
Adds the countdown panel variant.

Also: 
- removes bug where attributes are being added to panels in two places
- removes some whitespace and solves some indentation issues
- refactors timeout.js to allow it to be used by the countdown panel and the countdown modal
- updates docs

### How to review
- Check new tests cover all functionality
- Check tests pass
- Check new panel variant works as expected
